weddingbells/src/actions/index.js

```diff
@@ -11,10 +11,12 @@ export const SIGNUP_START = "SIGNUP_START";
 export const SIGNUP_SUCCESS = "SIGNUP_SUCCESS";
 export const TOGGLE_AUTH_MODAL = "TOGGLE_AUTH_MODAL";
 
+const BACKEND_API_URL = process.env.REACT_APP_BACKEND_API_URL;
```

Comment:
In case someone forgets to set this variable, make it work in development using `localhost`

```suggestion
const BACKEND_API_URL = process.env.REACT_APP_BACKEND_API_URL || http://localhost:5000;
```